deploy/quickstart-k8s.yaml

```diff
@@ -1,164 +0,0 @@
-apiVersion: v1
```

Comment:
Will the release pipeline be updated in the scope of this PR or in a separate one? It references the deleted quickstart-k8s

Reply:
Yes, I have it in mind. I planned to do it in a separate PR, so its easy to review. This one is already pretty big 🙈

Reply:
Okay nevermind, the change was small, so I included it in this PR.. Can you please have another look and let me know if that is all that's needed for the release?

Reply:
Great, thanks Mayank, looks good!